src/transformers/data/data_collator.py

```diff
@@ -213,6 +213,8 @@ def _collate_batch(examples, tokenizer):
 
     # Creating the full tensor and filling it with our data.
     max_length = max(x.size(0) for x in examples)
+    if pad_to_multiple_of is not None and (max_length % pad_to_multiple_of != 0):
+        max_length = ((max_length // pad_to_multiple_of) + 1) * pad_to_multiple_of
```

Comment:
@sgugger I was considering: typically max model token lengths are already multiples of 8 so it should not be a problem in practice, but in principle the pad to 8x in this method could make the `max_length` bigger then the max model length if that has some strange value. Do you think it's worth adding a check/truncation for that?

Reply:
I don't think we would ever get a problem with it for the reason you mention (all models have max length that are multiple of 8) and if it did happen it's easy for the user to set a lower max length in the scripts.